core/src/toga/app.py

```diff
@@ -183,12 +184,16 @@ def __init__(
             [`importlib.metadata`][]. If not provided, the following will be tried in
             order:
 
+            #. If the `app_id` argument was provided, its last segment will be used. For
+               example, an `app_id` of `com.example.my-app` would yield a distribution
+               name of `my-app`.
```

Comment:
This isn't rendering as a list.

Reply:
This seems to be a bug/limitation of mkdocstrings when used with Sphinx-style docstrings. For some reason, the parser isn't reading any content that includes newlines. 

I've logged this as mkdocstrings/mkdocstrings#808; in the meantime, I've reworked the text to not require bullet points.